sleuth_crawler/scraper/scraper/spiders/parsers/utils.py

```diff
@@ -19,7 +19,7 @@ def strip_content(data):
         return lines
```

Comment:
Just realized that you have `lines.append(line.strip())`, but you already stripped the line so there's no need to call strip() again.

Reply:
ay good catch